Assets/Scripts/UI/Console.cs

```diff
@@ -32,7 +32,7 @@ public class Console : PauseMenuWindow
 
 		static readonly Dictionary<LogType, Color> s_logTypeColors = new Dictionary<LogType, Color>
 		{
-			{ LogType.Assert, Color.red },
+			{ LogType.Assert, Color.green },
```

Comment:
why would you change this ? the color in Editor's console is red.

Reply:
Agree